exampleCourse/questions/element/truthTable/question.html

```diff
@@ -0,0 +1,33 @@
+<pl-question-panel>
+<p>
+What is the resulting values of the expressions
+<pl-code language="python">y == A or B</pl-code>
+<pl-code language="python">y == C and D</pl-code>
+<pl-code language="python">y == D and E</pl-code>
+<pl-code language="python">y == X or Y</pl-code>
+given the values in the following truth tables?
+</p>
```

Comment:
I can see how this can be confusing. After reworking the example question to have descriptions, I think that the question would be better off without this initial prompt and instead should have a short description of the element like in other example questions for elements anyways.